src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
```

Comment:
```suggestion
Add the `js-elastic-textarea` CSS class to a `<textarea>` element\* to have it expand as the user types more than will fit.
```

Reply:
Closing this in favor or @Paul-Hebert's [suggestions](https://github.com/cloudfour/cloudfour.com-patterns/pull/716/files#r428219468).